### ISSUE TITLE
Pre-train in gdsr/method lua scripts

### ISSUE THREAD

**Issue by eturner303** [griegler/primal-dual-networks#1]
Greetings!  Thanks for creating this project and writing both the Dual Primal and ATGV papers.  I'm attempting to replicate the results in the dual-primal paper and have run into a point of confusion:

Section 3.3 of the dual-primal-networks paper describes a pre-training procedure where the FCN is trained for 25 epochs and then FCN+PDN are jointly trained for another 10 epochs.  

Looking through the gdsr/method lua implementations, it appears the PDN joint training process is loading epoch 30 of the FCN, not epoch 25.  Is this intentional or a typo?

Just wanted to ensure I replicate your training process properly vs making assumptions based on either the paper or the GitHub code.  Thanks!

**Comment by griegler** [comment]
For the paper I used a small subset of the training data as validation set, i.e. ~25 samples that I did not use for training and used the network that performed best on this validation set. Therefore, neither of them are fully correct. The numbers reported in the paper were the maximum number of epochs. Sorry for the inaccuracy.

**Comment by 980380446** [comment]
could you please tell me how to run the code?thanks.
